### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Keyboard:Keyboard focus is lost after Installed toolbar.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52610]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : WIN10
**Additional Details:**
Keyboard,MAS2.4.3,ScreenReader-MAS1.3.1,MACOS High Sierra 10.13.5 Beta (17F70a)
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to Installed Toolbar.
4. Check the keyboard navigation.

**Actual:**
Keyboard focus is lost after the Installed Toolbar and NVDA also reads as extension treeview which not showing on the screen.
Same issue observed in MAC also.
**Expected:**
Focus should go to Recommended toolbar after the Installed Toolbar.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User unable to know where is the focus.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**

[[Accessibility]A11y_VSCode_ViewExtension_Keyboard1.pptx](https://github.com/Microsoft/vscode/files/2123745/Accessibility.A11y_VSCode_ViewExtension_Keyboard1.pptx)
[[Accessibility]A11y_VSCode_ViewExtension_NVDA.pptx](https://github.com/Microsoft/vscode/files/2123782/Accessibility.A11y_VSCode_ViewExtension_NVDA.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52596
